src/patternfly/components/Pagination/pagination.scss

```diff
@@ -1,6 +1,11 @@
 $pf-c-pagination--breakpoint-map: build-breakpoint-map();
+$pf-c-pagination--variable-map: build-spacer-map("none", "md", "xl");
 
 .pf-c-pagination {
```

Comment:
We should also add support for `.pf-m-page-insets`. You can look at the tabs component for a good example of how to do that. Here's the idea:

1. define the base page inset var and xl viewport inset var https://github.com/patternfly/patternfly/blob/1ead5fd152aa9ff95020036e1c8708cf01328205/src/patternfly/components/Tabs/tabs.scss#L15-L17
2. add an xl media query and update the base inset var to be set to the xl inset var
  https://github.com/patternfly/patternfly/blob/1ead5fd152aa9ff95020036e1c8708cf01328205/src/patternfly/components/Tabs/tabs.scss#L175-L178
3. then add a `.pf-m-page-insets` block where you set the padding to the page inset var.
  https://github.com/patternfly/patternfly/blob/1ead5fd152aa9ff95020036e1c8708cf01328205/src/patternfly/components/Tabs/tabs.scss#L467-L469